guide/ops/troubleshooting/deployment.md

```diff
@@ -143,3 +143,25 @@ start tasks are completed).
 
 See the [overview](overview.html) for where to find additional information, especially the section on
 "Entity's Error Status".
+
+## Invalid packet error
+
+If you receive an error message similar to the one below when provisioning a Centos VM, you can workaround the issue by explicitly setting the user that AMP should use to login to the VM.
```

Comment:
Ubuntu machines print a similar message on bluebox - better not restrict to centos.